src/main/java/net/masterthought/cucumber/json/Match.java

```diff
@@ -1,13 +1,55 @@
 package net.masterthought.cucumber.json;
 
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.LinkedList;
+import java.util.List;
+
 public class Match {
 
     // Start: attributes from JSON file report
-    private final String location = null;
+    private final JsonElement location = null;
     // End: attributes from JSON file report
 
     public String getLocation() {
-        return location;
+        if (location==null){
+            return "";
```

Comment:
gson leaves location as null when this element is not found.
We have to return empty string, because of usage of .getLocation() without null checking in net/masterthought/cucumber/json/Element.java:174